<launch>
    <node name="fgm_pp_v2" pkg="fgm_pp_gnu" type="sim.py" output="screen">
        <param name="robot_scale" value="0.3302" />
        <param name="planner" value="FGM_PP_V2" />
        <param name="param_path" value="sim_params.yaml" />
    </node>
</launch>